presto-spi/src/main/java/com/facebook/presto/spi/ConnectorTableMetadata.java

```diff
@@ -97,11 +100,19 @@ public Optional<String> getComment()
         return comment;
     }
 
-    public List<TableConstraint<ColumnHandle>> getTableConstraints()
+    public List<TableConstraint<String>> getTableConstraints()
     {
         return tableConstraints;
     }
 
+    public List<TableConstraint<ColumnHandle>> getTableConstraintsWithColumnHandles()
```

Comment:
Do we now have two methods because sometimes we need the column name, and sometimes we need the column handle? Can we just create a holder class for this purpose if that's the case?

Reply:
Added new class TableConstraintsHolder